reconcile/utils/terrascript_client.py

```diff
@@ -3624,6 +3625,16 @@ def populate_tf_resource_route53_zone(self, resource, namespace_info):
         zone_tf_resource = aws_route53_zone(zone_id, **values)
         tf_resources.append(zone_tf_resource)
 
+        # outputs
+        # zone id
+        output_name_0_13 = output_prefix + '__zone_id'
+        output_value = f'${{{zone_tf_resource.zone_id}}}'
+        tf_resources.append(Output(output_name_0_13, value=output_value))
+        # name servers
+        output_name_0_13 = output_prefix + '__name_servers'
+        output_value = f'${{{zone_tf_resource.name_servers}}}'
```

Comment:
reusing variables is a source of confusion.

Reply:
who's going to be the unlucky person who gets to refactor this entire file? :)

Reply:
always assume it is future you :)

Reply:
damn! ok, at least we agree it is _future_ me :)